solver/src/main.rs

```diff
@@ -253,13 +256,17 @@ async fn build_amm_artifacts(
                     .await
                     .expect("couldn't load deployed sushiswap router");
                 let pair_provider = Arc::new(SushiswapPairProvider { factory });
-                res.push(UniswapLikeLiquidity::new(
+                res.push(Box::new(UniswapLikeLiquidity::new(
                     IUniswapLikeRouter::at(&web3, router.address()),
                     pair_provider.clone(),
                     settlement_contract.clone(),
                     base_tokens.clone(),
                     web3.clone(),
-                ));
+                )));
+            }
+            BaselineSources::BalancerV2 => {
+                unimplemented!();
```

Comment:
uber-nano-nit: In general, prefer `todo` for blocks that are not yet implemented but you intend to implement. From [the docs]():
> The difference between `unimplemented!` and `todo!` is that while `todo!` conveys an intent of implementing the functionality later and the message is “not yet implemented”, `unimplemented!` makes no such claims. Its message is “not implemented”. Also some IDEs will mark `todo!`s.